[SM-515] Add reference to the use of secrets (#47)
mulesoft/docs-service-mesh: modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

. Verify the <<adapter_status, adapter status,>> using the following command:
`asmctl adapter list`

> == Reference Kubernetes Secret

Reviewer: ditto previous edits to same text; does it need to be repeated in both locations?

Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.

*Task Prerequisites*

> You must have the required permission to create the Kubernetes secret.

Reviewer: SUGGESTION:
Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials. You must have the required permission to create the Kubernetes secret.

. Verify the <<adapter_status, adapter status,>> using the following command:
`asmctl adapter list`

== Reference Kubernetes Secret

Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.

*Task Prerequisites*

> You must have the Devops or Admin role or permission to create the Kubernetes secret.

Author: ```suggestion
You must have permission to create the Kubernetes secret.
```

Reviewer: ditto previous questions about role or permission names

Reviewer 2: Done! Thanks!

> //[PA: Rodrigo--Are these the exact nams of the roles or permissions?]

Author: ```suggestion
```

Reviewer 2: Done! Removed!

To store your user credentials in Kubernetes secrets:

> . In a `.yaml` file, create a secret in which to store the sensitive information, for example, `bookinfo-service-mesh-adapter-config.yaml`:

Reviewer: ditto
```suggestion
. In a YAML file, create a secret in which to store the sensitive information, for example, `bookinfo-service-mesh-adapter-config.yaml`:
```

Reviewer 2: Done! Thanks!

+
[source,text,linenums]
----
apiVersion: v1
kind: Secret
metadata:
  name: <secret name> # example: bookinfo-service-mesh-adapter-config
  namespace: <namespace name> # example: default
type: Opaque
stringData:
  config: |
    {
      "clientId": "<replace-with-clientId>",
      "clientSecret": "<replace-with-clientSecret>"
    }
----
. Apply the secret: 
+
`kubectl apply -f bookinfo-service-mesh-adapter-config.yaml`
+

> If the secret is correctly applied, a "secret/bookinfo-service-mesh-adapter-config created" message is displayed.

Reviewer: font
```suggestion
If the secret is correctly applied, a 'secret/bookinfo-service-mesh-adapter-config created' message is displayed.
```

Reviewer 2: Done! Thanks!
